packages/beacon-node/src/network/gossip/gossipsub.ts

```diff
@@ -99,7 +99,7 @@ export class Eth2Gossipsub extends GossipSub {
       fanoutTTL: 60 * 1000,
       mcacheLength: 6,
       mcacheGossip: 3,
-      seenTTL: 550 * GOSSIPSUB_HEARTBEAT_INTERVAL,
+      seenTTL: config.SECONDS_PER_SLOT * SLOTS_PER_EPOCH * 2,
```

Comment:
Would adding a comment pointing to the specs make sense here?

Reply:
this is a pretty straightforward value which can be easily found in the spec
also EIP-7045 happens in a couple of places not just this one

Reply:
this should be in `ms` not `second`, fixed in #6607